frontend/src/components/about/Platform.vue

```diff
@@ -49,13 +49,20 @@
       </p>
 
       <h3 id="citation" class="title is-3 pt-6">Citation</h3>
-
-      <h4 class="title is-4">Latest publication</h4>
       <p>
         If you use Metabolic Atlas in your work, or any of the animal models, please cite our latest
         publication. In addition, if you use Human-GEM, please also cite the corresponding
         publication.
       </p>
+      <h4 id="citation-v3" class="title is-4">3.0</h4>
+      <p>Article under consideration.</p>
+
+      <h4 id="citation-v2" class="title is-4">2.0</h4>
+      <p>
+        Version 2.0 of the <i>Metabolic Atlas</i> was published in 2021 together with a PNAS
+        publication. If you have used <i>Metabolic Atlas</i> version 2.0 to 2.6 in your work, please
```

Comment:
I get why we are writing this, but I'm not sure how users will know if they used 2.0-2.6, 1.0-1.7 etc. or not. We currently don't support visiting previous version of the site either. Maybe we can keep this and users can just ask if they don't know how to cite, but I wanted to raise that this information is likely a bit confusing.

Reply:
I agree with Ingrid that it is confusing for people to cite old version of MetAtlas since we don't support it right now. It's probably better to point out a specific claim or finding of the publication to be cited than mention the version of the website.

Reply:
Good point! Updated in 408c7a2, please check if you find this verison better.